AUTHORS

```diff
@@ -3021,5 +3020,125 @@ David Gilbertson <gilbertson.david@gmail.com>
+Hachimi Aa (Sfeir) <hachimi.aa@sfeir.com>
+Daniel Estiven Rico Posada <danielrico.posada@gmail.com>
+Andrés M. Gómez <andres86mauricio@hotmail.com>
+Nick Kreeger <nick.kreeger@gmail.com>
+Moni <40552237+NotMoni@users.noreply.github.com>
+Dave Vandyke <kzar@kzar.co.uk>
+Agustin Daguerre <agustin.s.daguerre@gmail.com>
+Jesus Hernandez <jesusbeckan@gmail.com>
+龙腾道 <LongTengDao@LongTengDao.com>
+John Kleinschmidt <jkleinsc@github.com>
+Cecchi MacNaughton <cecchi.macnaughton@gmail.com>
+Reşat SABIQ <haqer@gmx.fr>
+daemon1024 <barun1024@gmail.com>
+Qinhui Chen <lianxuify@gmail.com>
+William Bonawentura <sla100@poczta.fm>
+Vadzim Zieńka <v@vadzim.info>
+David Daza <03dazal@gmail.com>
+雨夜带刀 <chenmnkken@gmail.com>
+karan singh virdi <karan1205singh@gmail.com>
+William Armiros <54150514+willarmiros@users.noreply.github.com>
+Matt Kulukundis <matt.fowles@gmail.com>
+Edward Elric <sasuke688848@gmail.com>
+Prosper Opara <oparaprosper79@gmail.com>
+Eileen <eileenlu0306@gmail.com>
+Ishaan Jain <37652866+ishaanjain1898@users.noreply.github.com>
+Christopher Beeson <christopher@bl-ue.com>
+thecodrr <thecodrr@protonmail.com>
+Chris Holland <chrisholland3553@gmail.com>
+Ranjan Purbey <ranjan.purbey@gmail.com>
+白一梓 <yunnysunny@gmail.com>
+John Leidegren <john@leidegren.se>
+Karol Walasek <coreconviction@gmail.com>
+osher <osher.filter@gmail.com>
+szTheory <szTheory@users.noreply.github.com>
+Jonathan Buhacoff <jonathan@buhacoff.net>
+Paolo Insogna <paolo@cowtech.it>
+Richard Townsend <Richard.Townsend@arm.com>
+Dan Fabulich <dan@fabulich.com>
+Deep310 <55121371+Deep310@users.noreply.github.com>
+Dominykas Blyžė <hello@dominykas.com>
+Wenning Zhang <1527841714@qq.com>
+shisama <shisama07@gmail.com>
+Kirill Shatskiy <shaackiy@gmail.com>
+Ben Bucksch <ben.bucksch@beonex.com>
+Eli Schwartz <eschwartz@archlinux.org>
+Maciej Kacper Jagiełło <maciej@jagiello.it>
+Tom Nagle <tomanagle@gmail.com>
+rickyes <ives199511@gmail.com>
+sapics <gv.nishino@gmail.com>
+Sagar Jadhav <sagarj.jadhav23@gmail.com>
+Dennis Ameling <dennis@dennisameling.com>
+Todd Short <tshort@akamai.com>
+Lucas Pardue <lucaspardue.24.7@gmail.com>
+Pragyan Das <pragyandas@msn.com>
+Shakil-Shahadat <shakil.regi@gmail.com>
+wenningplus <zhangwenning@videopls.com>
+WilliamConnatser <43946230+WilliamConnatser@users.noreply.github.com>
+Benjamin Kaduk <bkaduk@akamai.com>
+Tatsuhiro Tsujikawa <tatsuhiro.t@gmail.com>
+Nathan Blair <engineer.nblair@gmail.com>
+patr0nus <dk4rest@gmail.com>
+Alec Davidson <davidson9076@gmail.com>
+falguniraina <48027052+falguniraina@users.noreply.github.com>
+Rodion Abdurakhimov <rodion.arr.web@gmail.com>
+Anentropic <ego@anentropic.com>
+Saleem <hamids2@asme.org>
+Julien Poissonnier <julien@caffeine.lu>
+zombieleet <zombieleetnca@gmail.com>
+Nikola Glavina <nikola.glavina@student.um.si>
+Johannes Schöpp <schoepp@rz.uni-frankfurt.de>
+Francisco Ryan Tolmasky I <tolmasky@gmail.com>
+Ye-hyoung Kang <keepyourhonor@gmail.com>
+Myles Borins <mylesborins@github.com>
```

Comment:
@MylesBorins which is your preferred address?

Reply:
github.com for the time being